schemas/vertices/taxa.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "description": "Biological taxonomic node (class, order, family, genus, species, organism)",
+  "required": ["_key", "name"],
+  "properties": {
+    "_key": {
+      "type": "string",
+      "example": "haloferax_volcanii",
+      "description": "Normalized name (lower-cased and snake-cased)"
+    },
+    "name": {
+      "type": "string",
+      "example": "Haloferax Volcanii"
```

Comment:
In future I'd like to also encode the scientific lineage (class, order, family, genus, species, organism) as an array. If take from NCBI there's also a numerical id I think

Reply:
We could put that in an attribute in the genome document. Or are you thinking of a separate vetex?